src/algebra/group/hom.lean

```diff
@@ -81,6 +81,14 @@ lemma to_fun_eq_coe (f : M →* N) : f.to_fun = f := rfl
 @[simp, to_additive]
 lemma coe_mk (f : M → N) (h1 hmul) : ⇑(monoid_hom.mk f h1 hmul) = f := rfl
 
+@[to_additive]
+theorem congr_fun {f g : M →* N} (h : f = g) (x : M) : f x = g x :=
```

Comment:
Should we define `congr_fun'` for anything that has a `coe_to_fun`? Then we're just done with all of them in one go (and we could even have sweet notation?)

Reply:
I think the problem is that just knowing that something `has_coe_to_fun` doesn't tell you what the domain type is (the domain is a field of `has_coe_to_fun`, rather than a parameter!) So it's impossible to state the lemma at that level of generality...?